src/main/java/org/skywalking/apm/mock/collector/service/MockApplicationRegisterService.java

```diff
@@ -12,13 +14,14 @@
 import org.skywalking.apm.mock.collector.entity.RegistryItem;
 import org.skywalking.apm.mock.collector.entity.ValidateData;
 
+import static org.skywalking.apm.mock.collector.service.Sequences.APPLICATION_MAPPING;
+import static org.skywalking.apm.mock.collector.service.Sequences.SERVICE_SEQUENCE;
```

Comment:
Look like wrong concept. Should name service_mapping and endpoint_sequence?

Reply:
Yes. they must use the single sequence. The mock collector support both 5.x and 6.x. such as `header-scenario`.